.jenkins/pytorch/test.sh

```diff
@@ -34,6 +34,15 @@ if [[ "$BUILD_ENVIRONMENT" == *asan* ]]; then
     (cd test && ! python -c "import torch; torch._C._crash_if_aten_asan(3)")
 fi
 
+export ATEN_DISABLE_AVX=0
```

Comment:
These shouldn't be set at all. Our dispatch code only checks whether the environment variables has been set, this is similar to USE_MKLDNN etc. I'll also send out a PR soon that switches this from ATEN_DISABLE_AVX to USE_AVX etc., but for now we should just get this landed so we can be sure it's tested.